README.md

```
@@ -1,43 +1,80 @@
 
-Personal Claude Code plugin: a GitHub issue → PR workflow, factored into three skills.
+_Personal AI agent plugin: a collection of agent-agnostic skills_.
+
+- GitHub issue → PR workflow, factored into separate skills.
+- Obsidian
+- Graphify usage
+- etc.
 
 ## Install
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)